.github/CODEOWNERS

```diff
@@ -107,6 +107,9 @@
 # PRLabel: %Cognitive - Metrics Advisor
 /sdk/metricsadvisor/                                                 @xiangyan99 @kristapratico
 
+# PRLabel: %Cognitive - Translator
+/sdk/documenttranslation/                                            @mohamedshabanofficial @kristapratico
```

Comment:
`/sdk/documenttranslation/ @kristapratico @mohamedshabanofficial`
change the order :)

Reply:
I don't think the order matters, but sure I can swap us.